.travis/custom.sh

```diff
@@ -3,13 +3,14 @@
 
 set -e
 
-ME=$(basename $0)
-SCRIPTDIR=$(readlink -f $(dirname $0))
+#uncomment if you use $ME - otherwise set in utils.sh
```

Comment:
This pr sets ME in utils.sh [0]. So we could just get rid of setting ME in each shell script? (not just from the scripts ME is not used, but is being used such as runcoveralls.sh, runflake8.sh, etc.?)
[0] - https://github.com/linux-system-roles/network/pull/261/files#diff-0dc30956d7844f3c69f23e3fd299cee0R197

Reply:
@nhosoi yes, we could probably get rid of `ME` in each shell script.  However, I didn't want to do this in this PR, since it will require some manual inspection to see if any of the custom scripts use `ME`, or did not source utils.sh.  We can do that in another PR.